src/main/webapp/js/common.js

```diff
@@ -161,6 +161,17 @@ $(document).on('ajaxComplete ready', function() {
     if (isTouchDevice()) {
         $tooltips.tooltip('disable');
     }
+    
+    /**
+     * Underlining all span elements with tool-tips except for 
+     * ones without a text value.
+    */
+    $('span[data-toggle="tooltip"]').each(function() {
+        textValue = $(this).text().replace(/\s/g, '')
```

Comment:
Perhaps we can add a comment here explaining why we test for empty string. Alternatively, it'll be clearer to explicitly filter out elements that ought to have tooltip but not underline, e.g. `$('span[data-toggle="tooltip"]').not('.glyphicon')...`